Add introduction file

kaylabranch opened the pull request.

PeterBell committed:
Merge pull request #2 from githubteacher/master

Most Senseis should be respected

Commit:
Initial kill list

PeterBell committed:
Merge pull request #3 from deadlyvipers/kill_list

Initial kill list

Commit:
We dont need to kill Budd

kaylabranch committed:
Add introduction file

codeschool-kiddo commented on introduction.md:
```diff
@@ -0,0 +1 @@
+This file contains an introduction.
```
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.

kaylabranch committed:
Added path

kaylabranch committed:
Adding skills

kaylabranch committed:
Try 2

kaylabranch committed:
Revision

codeschool-kiddo committed:
Added new rule about coffee

kaylabranch committed:
Fixing spelling errors

kaylabranch committed:
Merge pull request #2 from kaylabranch/new_rules

Too many people are spilling coffee

kaylabranch committed:
Merge branch 'deadly_skills'

kaylabranch committed:
Replacing names
Unused code
Functions with side effects
Unformatted code
Repeated code blocks
Bad variable names

kaylabranch committed:
Merge branch 'release_branch_1.0'

kaylabranch committed:
This commit fixes #3

kaylabranch committed:
Merge pull request #4 from kaylabranch/kill_list

This commit fixes #3

kaylabranch committed:
Updating readme

kaylabranch committed:
Merge branch 'kill_list'
